db/migrate/20150114194931_create_answers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAnswers < ActiveRecord::Migration
+  def change
+    create_table :answers do |t|
+      t.belongs_to :question
+      t.belongs_to :user
```

Comment:
this gave us "Index name 'index_answers_on_user_id' on table 'answers' already exists" and DB rollback, until we got rid of the line "add_index :answers, :user_id"